docs/backend/auth-twitter.md

```diff
@@ -89,8 +89,6 @@ application.
     ```text
     API key -> TWITTER_KEY
     API secret key -> TWITTER_SECRET
-    Access Token -> TWITTER_ACCESS_TOKEN
```

Comment:
Note that the corresponding image was not updated. Unlike the Gitdocs documentation indicates it is not in `docs/.static`, so I wasn't sure how it gets uploaded. We can leave as is (though it might be slightly confusing) or someone enlightens me on how to get it updated.

Reply:
I usually upload an image to a Github issue or comment form and then copy the link :D

Reply:
Haha, ok, that seems a bit random 😄 Anyway, I'll add an updated image tomorrow (or maybe just crop the current one).

Reply:
Updated screenshot:

<img width="1440" alt="Screen Shot 2020-01-14 at 16 03 31" src="https://user-images.githubusercontent.com/47985/72329507-72d30a00-36e7-11ea-83ac-ebea5d41ba39.png">